src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-height: 358px;
+  max-width: 166px;

```

Comment:
remove height and width

Reply:
when I remove height and width I can't pass the test

Reply:
add width 200px and use box-sizing on card to change the default calculation of element size
remove height